docs/openlaw-object/README.md

```diff
@@ -0,0 +1,1857 @@
+# Openlaw Object
+
+The `Openlaw` object defined in Openlaw.scala compiles to JavaScript and is an interface in the OpenLaw protocol to interact directly with an agreement and its contents, including its various variable types. The object methods are categorized below.
+
+## Template / Contract
+
+### compileTemplate
+
+TODO.
+
+```
+compileTemplate(
+  text: String
+): js.Dictionary[Any]
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `text` | `string` | **Required.** TODO. |
+
+Example
+
+```
+
+```
+
+**Response**
+
+Returns `js.Dictionary[Any]` - TODO.
+
+Example
+
+```
+
+```
+
+### execute
+
+TODO.
+
+```
```

Comment:
VuePress can highlight most popular language syntax, so if you change the start of the block to hint language via:

    ```js

Then it should highlight! Yay!

Reply:
oh nice. thanks!